packages/app/test/cypress/integration/80-notification/confirm-notification.spec.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+context('Confirm notification', () => {
+  const ssPrefix = 'confirm-notification-';
+
+  let connectSid: string | undefined;
+
+  before(() => {
+    // login
+    cy.fixture("user-admin.json").then(user => {
+      cy.login(user.username, user.password);
+    });
+    cy.getCookie('connect.sid').then(cookie => {
+      connectSid = cookie?.value;
+    });
+  });
+
+  beforeEach(() => {
+    if (connectSid != null) {
+      cy.setCookie('connect.sid', connectSid);
+    }
+  });
```

Comment:
- before / beforeEach
I think these code could be more compact. Please refer to acces-to-spec.ts

Reply:
Removed `before` and use `beforeEach` instead